orange/src/main/java/com/hartwig/hmftools/orange/report/tables/GainLossTable.java

```diff
@@ -42,7 +42,7 @@ public static Table build(@NotNull String title, float width, @NotNull List<Purp
             table.addCell(Cells.createContent(gainLoss.chromosome() + gainLoss.chromosomeBand()));
             table.addCell(Cells.createContent(displayGene(gainLoss)));
             table.addCell(Cells.createContent(display(gainLoss.interpretation())));
-            table.addCell(Cells.createContent(String.valueOf(gainLoss.minCopies())));
+            table.addCell(Cells.createContent(String.format("%.1f", gainLoss.minCopies())));
```

Comment:
Is this functionally identical to rounding used in other places? Eg see BreakendTable:

```
private static final DecimalFormat SINGLE_DIGIT = ReportResources.decimalFormat("#0.0");
```

Reply:
I standardized all decimal formatting that referenced `decimalFormat` and used the new common method here.